.github/workflows/aws_deploy.yml

```diff
@@ -129,24 +129,46 @@ jobs:
           succeeded=$(aws ecs describe-tasks --cluster $cluster --tasks $id --query "tasks[0].stopCode == 'EssentialContainerExited' && to_string(tasks[0].containers[0].exitCode) == '0'")
           if [ $succeeded == true ]; then exit 0; else exit 1; fi
 
-      - name: Download service task definition
+      - name: Download app service task definition
         env:
           app_task_definition: ${{ inputs.aws_resource_prefix }}-app
         run: |
           aws ecs describe-task-definition --task-definition $app_task_definition --query taskDefinition > app-task-definition.json
 
-      - name: Update image ID
+      - name: Download sidekiq service task definition
+        env:
+          app_task_definition: ${{ inputs.aws_resource_prefix }}-sidekiq
```

Comment:
Do you think we should maybe rename this to sidekiq_task_definition to avoid confusion?

Reply:
Yes good spot! I'll update this 👍